### ISSUE TITLE
Svgcleaner improvement of file size

### ISSUE THREAD

**Issue by devrck** [simple-icons/simple-icons#1135]
Hi,
I've noticed that with the latest release of svgcleaner the svg size could go down for all icons with something around 5%. Is it worth the effort?

**Comment by devrck** [comment]
Hi @davidklebanoff,

SVGO and svgcleaner are similar so no need to switch between them I think at lease for now.

What svgcleaner did after I ran it on graphql.svg it stripped out the title and role="img" which saved some space but I believe you need it for accessibility, so that's not gonna fly.

Although it stripped out these parts I notice that maybe you want to use `aria-label="GraphQL"` instead of title which will save maybe 2% but maybe its not worth it.

**Comment by ericcornelissen** [comment]
I think using SVGO or svgcleaner, for our purposes, does not make a difference based on what I've read. The main reason I think we should stick with recommending contributors to use SVGO rather then svgcleaner is because SVGO (already) has a web UI, whereas svgcleaner requires a download (which requires more effort from new contributors).

I would support an update to the contributing guidelines stating that one can use svgcleaner as well, for example:

```markdown
### 3. Optimize The Icon

All icons in Simple Icons have been optimized with the [SVGO tool](https://github.com/svg/svgo). This can be done in one of two ways:

[...]

Alternatively you could use [svgcleaner](https://github.com/RazrFalcon/svgcleaner) or [its GUI](https://github.com/RazrFalcon/svgcleaner-gui)
```

* * *

> Although it stripped out these parts I notice that maybe you want to use aria-label="GraphQL" instead of title which will save maybe 2% but maybe its not worth it

If all we want is to support accessibility, that is a good suggestion. The main difference between `<title>` and `aria-label` (that I could find) is that the former also gives you the tooltip when you hover over the SVG. Arguably we do not want this, as the string we use for `<title>` is not helpful in most scenarios 🤔

**Comment by ericcornelissen** [comment]
tl;dr we may switch to `aria-label` in the future once SVG2 has been widely adopted but for now we have to stick with `<title>`

* * *

I tried to investigate **why** we're using `<title>` rather then `aria-label` but I was unsuccessful. Most icons had `<title>` added in #500 but that was in order to be in line with the contributing guidelines, which was originally added in #475 with mention of `<title>` already included. This pull request refers to #472, which has some more details about the accessibility standards behind our decisions, namely [the W3C's Accessibility Features of SVG article](https://www.w3.org/TR/SVG-access/) and [this CSS-Tricks article](https://css-tricks.com/accessible-svgs/) (which is a bit out of date).

From the W3C's article it seems that we should use `<title>` rather then `aria-label`, given that it only mentions the former. However, from [the W3C's SVG Accessibility API Mappings article]() (specifically [section 5](https://www.w3.org/TR/svg-aam-1.0/#mapping)) it is clear that one can use `aria-label` starting from SVG2:

> **WAI-ARIA support was formally introduced to SVG in Scalable Vector Graphics (SVG) 2 [SVG2], which allows ARIA attributes to be used in the default namespace.** This section defines how WAI-ARIA semantics are exposed to assistive technologies through platform Accessibility APIs and how SVG elements are mapped to Accessibility APIs based on WAI-ARIA. 

Unfortunately SVG2 most recent development was in October 2018 (see [this candidate recommendation](https://www.w3.org/TR/SVG2/)) so it is not yet standardized/widely supported.
